app/controllers/messages_controller.rb

```diff
@@ -1,12 +1,7 @@
-class MessagesController < ApplicationController
-
-    def index
-
-        #@message = Message..order("RANDOM()").first
-        # @message = Message.order("RAND()").limit(1)
-        @message = Message.limit(2).order("RANDOM()")
-        #json_response(@message)
-        render json: @message, status: :ok
-    end
-
-end
\ No newline at end of file
+class MessagesController < ApplicationController
+  def index
+    @message = Message.limit(2).order('RANDOM()')
+
+    render json: @message, status: :ok
+  end
+end
```

Comment:
- [ ] When I start the front-end server, I am greeted with the "No message found" which was strange because I tested the API with my own data and it was working perfectly.

![image](https://user-images.githubusercontent.com/86318284/210856924-75f0a2cc-77c8-426d-8762-318ee898226c.png)

On closer inspection in the networks tab, I realize that the reason why there is no data is that your API is prohibiting data from being sent to the front end because of [CORS (Cross-Origin Resource Sharing)](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS). Luckily for us, there is a solution to solving this using the rack-cors gem ```gem 'rack-cors'```. Kindly visit the [official docs](https://github.com/cyu/rack-cors) to learn how to set it up. It is a two-step process trust me.

![image](https://user-images.githubusercontent.com/86318284/210857575-994f0361-aaac-4efc-9bdd-fde2609220de.png)